src/config.rs

```diff
@@ -95,7 +95,7 @@ impl Cfg {
         );
         let dist_root = dist_root_server.clone() + "/dist";
 
-        let cfg = Cfg {
+        let cfg = Self {
```

Comment:
Is this a new lint I was previously unaware of?

Reply:
I just enabled [the lint](https://rust-lang.github.io/rust-clippy/master/index.html#use_self) in this PR.

Reply:
Aah I see.  This one is a *pedantic* lint and I'm not sure I completely care for it.  I won't say "no" this time, but it feels a bit overly pedantic for my liking.  Can make it harder to tell what something is if it's overused IMO.